src/platform/nrfconnect/wifi/WiFiManager.cpp

```diff
@@ -469,13 +507,52 @@ void WiFiManager::ConnectHandler(Platform::UniquePtr<uint8_t> data)
     }
 }
 
-void WiFiManager::DisconnectHandler(Platform::UniquePtr<uint8_t>)
+void WiFiManager::DisconnectHandler(Platform::UniquePtr<uint8_t> data)
 {
-    SystemLayer().ScheduleLambda([] {
+    CHIP_ERROR err = SystemLayer().ScheduleLambda([capturedData = data.get()] {
```

Comment:
We can't just capture the UniquePtr by move here?

Reply:
Our lambdas must be trivially copyable since they are passed via `ChipDeviceEvent`, which prevents capturing `UniquePtr`, right?

Reply:
Ah, ok, makes sense.